src/questionnaireitem_media_video.js

```diff
@@ -58,8 +77,25 @@ QuestionnaireItemMediaVideo.prototype._createMediaNode = function() {
     }
 
     this.videoNode = document.createElement('video');
+    this.videoNode.setAttribute("playsinline", "");
```

Comment:
Please add a comment above why this is done.
And please make a separate commit.

Reply:
Moved to new PR #55